File: app/src/main/java/com/example/themoviedb/popular_people_screen/popular_people_view/PopularPeopleViewInterface.java
package com.example.themoviedb.popular_people_screen.popular_people_view;

import com.example.themoviedb.popular_people_screen.popular_people_model.PopularPeople;

import java.util.List;

public interface PopularPeopleViewInterface {

    void setPerson(PopularPeople person);
    void settingAdapterInList();
    List<PopularPeople> getList();
    Boolean getSearchState();
    void notifyDataRemoved(int size);
}

File: app/src/main/java/com/example/themoviedb/popular_people_screen/popular_people_view/PopularPeopleViewInterface.kt
package com.example.themoviedb.popular_people_screen.popular_people_view

import com.example.themoviedb.popular_people_screen.popular_people_model.PopularPeople

interface PopularPeopleViewInterface {
    fun getList(): List<PopularPeople>?
    fun getSearchState(): Boolean?

    fun setPerson(person: PopularPeople)
    fun settingAdapterInList()
    fun notifyDataRemoved(size: Int)
}
